tests/src/test/scala/cats/tests/EitherSuite.scala

```diff
@@ -346,7 +346,7 @@ class EitherSuite extends CatsSuite {
   test("raiseOrPure syntax consistent with fromEither") {
     val ev = ApplicativeError[Validated[String, *], String]
     forAll { (fa: Either[String, Int]) =>
-      fa.raiseOrPure[Validated[String, *]] should ===(ev.fromEither(fa))
+      fa.liftTo[Validated[String, *]] should ===(ev.fromEither(fa))
```

Comment:
I just got bitten by leaving a deprecated method untested, so this makes me a little nervous.